In [38]:
import math

## Ejercicio 16: Trabajo en equipo.

En un grupo de 25 alumnos hay 13 chicas y 12 chicos. Se desean formar equipos de trabajo para realizar una investigación:

a) ¿Cuántos equipos de cinco personas pueden formarse?

Tenemos un total de 25 alumnos y si queremos realizar equipos de cinco personas cada uno usariamos la formula de las combinaciones, la cual es: 
$$
\begin{align}
    \binom{n}{k} = \frac{n!}{k!(n-k)!}
\end{align}
$$

Donde n = 25 y k = 5

$$
\begin{align}
    \binom{25}{5} &= \frac{25!}{5!(25-5)!}  &= 53130 \\ 
\end{align}
$$

In [39]:

equiposA = math.comb(25,5)

print(f'La cantidad de equipos que se pueden formar con 5 alumnos son: {equipos}')

La cantidad de equipos que se pueden formar con 5 alumnos son: 17160


b) ¿Cuántos equipos pueden formarse si debe haber una chica?

Primero debemos saber que si solo hay una chica en el equipo significa que los otros 4 integrantes seran chicos, sabiendo eso usamos la formula de las combinaciones para saber cuantas posibilidades hay de obtener a una chica en el equipo, y aparte saber cuantas hay de obtener a 4 chicos, para despues multiplicarlas entre si:
$$
\begin{align}
\binom{13}{1} = \frac{13!}{1!(13-1)!} = 13
\end{align}
$$
Son 13 posibilidades de que haya 1 chica en el equipo
$$
\begin{align}
\binom{12}{4} = \frac{12!}{4!(12-4)!} = 495
\end{align}
$$
Son 495 posibilidades de que haya 4 chicos en el equipo

$$
\begin{align}
    13 * 495 = 6435
\end{align}
$$

Ahora multiplicamos los resultados entre si para saber la cantidad de equipos posibles (los cuales son 6435)



In [40]:
equiposB = math.comb(13,1) * math.comb(12,4)
print(f'La cantidad de equipos que se pueden formar con 1 chica y 4 chicos son: {equipos}')

La cantidad de equipos que se pueden formar con 1 chica y 4 chicos son: 17160


c) ¿Cuántos equipos pueden formarse si debe haber dos chicas?

Siguiendo el mismo procedimiento del ejercicio anterior

Primero debemos saber que si solo hay dos chicas en el equipo significa que los otros 3 integrantes seran chicos, sabiendo eso usamos la formula de las combinaciones para saber cuantas posibilidades hay de obtener a dos chicas en el equipo, y aparte saber cuantas hay de obtener a 3 chicos, para despues multiplicarlas entre si:
$$
\begin{align}
\binom{13}{2} = \frac{13!}{2!(13-2)!} = 78
\end{align}
$$
Son 78 posibilidades de que haya 2 chicas en el equipo
$$
\begin{align}
\binom{12}{3} = \frac{12!}{3!(12-3)!} = 220
\end{align}
$$
Son 220 posibilidades de que haya 3 chicos en el equipo

$$
\begin{align}
    78 * 220 = 17160
\end{align}
$$
Ahora multiplicamos los resultados entre si para saber la cantidad de equipos posibles (los cuales son 17160)

In [41]:
equiposC = math.comb(13,2) * math.comb(12,3)
print(f'La cantidad de equipos que se pueden formar con 2 chicas y 3 chicos son: {equipos}')

La cantidad de equipos que se pueden formar con 2 chicas y 3 chicos son: 17160


d) Determina las probabilidades correspondientes a los incisos anteriores.

Para obtener las probabilidades correspondientes debemos saber que la probabilidad de obtiene dividiendo la cantidad de casos favorables entre la cantidad de casos totales:

$$
\begin{align}
    Probabilidad = \frac{Casos Favorables}{Casos Totales}
\end{align}
$$

Ya con eso podemos obtener las probabilidades correspondientes

Para el inciso b:

$$
\begin{align}
    Probabilidad = \frac{6435}{53130} =
\end{align}
$$

Y para el inciso c:

$$
\begin{align}
    Probabilidad = \frac{17160}{53130} =
\end{align}
$$

In [42]:
ProbabilidadB = equiposB/equiposA
print(f'La probabilidad de que salga un equipo con una chica y cuatro chicos es de: {round(ProbabilidadB*100,2)} %')
ProbabilidadC = equiposC/equiposA
print(f'La probabilidad de que salga un equipo con dos chicas y tres chicos es de: {round(ProbabilidadC*100,2)} %')


La probabilidad de que salga un equipo con una chica y cuatro chicos es de: 12.11 %
La probabilidad de que salga un equipo con dos chicas y tres chicos es de: 32.3 %
